src/cpu/x64/jit_copy_1d.hpp

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
```

Comment:
(random spot)

Hi @CaoZhongZ, thank you for the PR! 

Could you please post performance improvements you got with this it (and relevant data such as shapes / cases / data types / platforms you were testing)?
It makes sense to update the description of the PR with this information.